torch/jit/__init__.py

```diff
@@ -581,9 +582,16 @@ def __getattr__(self, attr):
         return self.module._get_method(attr)
 
 
-def script(fn, optimize=True, _frames_up=0):
+def weak_script(fn):
+    fn._jit_is_weak_script = True
+    return fn
+
+
+def script(fn, optimize=True, _frames_up=0, is_weak=False):
```

Comment:
Can you please make the last parameter `_is_weak` to emphasize that it's internal?